client/src/Components/Issue/InputComment.js

```diff
@@ -51,7 +51,9 @@ const InputComment = () => {
     const filePath = result.data;
     const text = `![images](${filePath})\n`;
     const inputContentElement = document.querySelector('#inputContent');
```

Comment:
지금 봤는데 querySelector가 있네요? 없이 할 수 있지 않을까요? `useRef` 라든지!

Reply:
그러게요! 수정하는게 좋아보이긴하네요 🤔

Reply:
`useRef`가 리액트에서 지원하는 것일까요?
공부해봐야겠네요.🤔

Reply:
commit 69b416d
useRef를 사용하여 DOM에 접근하도록 바꿔봤는데 querySelector가 아닌 useRef를 사용함으로써 얻는 이점에는 무엇이 있을까요?🤔

Reply:
@Park-SooYeon 
[참고자료!](https://blog.cloudboost.io/using-ref-in-react-a832fa3a5125) `useRef`는 특정 스코프 내에서 검색을 하고 querySelector는 전체 dom을 탐색하는 것 같네요.

> As we have heard lot of times earlier, everything in React is a component. And each component works independently and they have self-contained code. But global selectors don’t search just on your component, they search at global document level, and fetches the dom that may belong to some other component. And this will lead to lots of unpredictable bugs.